src/Compiler/Service/service.fs

```diff
@@ -209,7 +206,6 @@ type FSharpChecker
             ?parallelReferenceResolution: bool,
             ?captureIdentifiersWhenParsing: bool,
             ?documentSource: DocumentSource,
-            ?useSyntaxTreeCache: bool,
```

Comment:
This is binary breaking change, right? I wonder if we're fine breaking FSAC/Rider

Reply:
We build FCS ourselves for the Rider plugin and we don't use this parameter, so it should be fine for us.

Reply:
FSAC sets it, indeed.
https://github.com/ionide/FsAutoComplete/blob/258fbc55053f585ea6ebd81a7dc10e4f3573aff6/src/FsAutoComplete.Core/CompilerServiceInterface.fs#L94
It would require some coordination to fully remove it.

Reply:
I'll keep the surface area in case of any unforeseen consequences.

Reply:
We have binary breaking changes with basically every FCS anyway, it doesn't matter so much

Reply:
Cool, so let's clean this up fully.